capstone/src/main/resources/static/event.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <title> Create an Event </title>
+    <script src="script.js"> </script>
+    <link rel="stylesheet" href="event.css">
+  </head>
+  
+  <body>
+
+    <div class="menu">
+      <nav>
+        <a href="map.html"> Campus Map </a>
```

Comment:
Minor: this link shows up as dark purple on dark grey background for me; a bit hard to read.

Reply:
I will fix that